controllers/remote/flags.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remote
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+var (
+	restConfigTimeout time.Duration
+	restConfigQPS     float32
+	restConfigBurst   int
+)
```

Comment:
Is it possible for the variables be initialized here rather than in `init()`? I feel like this is marginally more straightforward.
```suggestion
var (
	restConfigTimeout time.Duration = defaultClientTimeout
	restConfigQPS     float32       = 20
	restConfigBurst   int           = 30
)
```